etor/build.gradle

```diff
@@ -53,3 +53,12 @@ jacocoTestReport {
 tasks.named('test') {
     useJUnitPlatform()
 }
+
+sourceSets {
+    main {
+        resources {
+            // Include the mock_credentials folder as part of the resources
+            srcDir '../mock_credentials'
+        }
+    }
+}
```

Comment:
I'd move this to the `shared` sub-project and delete the other secrets that are already stored in `/shared/src/main/resources/`.